build/storage/tests/it/ptf_tests/system_tools/const.py

```diff
@@ -12,3 +12,53 @@
 DEFAULT_HOST_TARGET_SERVICE_PORT_IN_VM = 50051
 DEFAULT_MAX_RAMDRIVE = 64
 DEFAULT_MIN_RAMDRIVE = 1
+
+FIO_RANDRW = {
+    "rw": "randrw",
+    "runtime": 1,
+    "numjobs": 1,
+    "time_based": 1,
+    "group_reporting": 1
+}
+FIO_RANDREAD = {
+    "rw": "randread",
+    "runtime": 1,
```

Comment:
I see that we have many duplications here
let's try to extract the common parameters in a separate dict
```
FIO_COMMON = {
    "runtime": 1,
    "numjobs": 1,
    "time_based": 1,
    "group_reporting": 1
}

FIO_RANDRW = {
    "rw": "randrw",
    **FIO_COMMON 
}

FIO_RANDREAD = {
    "rw": "randread",
     **FIO_COMMON 
}

# and the rest
```

Also I'd put them directly in the test file since they are fio test related dictionaries.

Reply:
changed